src/pages/MapViewer/components/Toolbox/components/ToolboxRow/index.tsx

```diff
@@ -21,9 +21,9 @@ const ToolboxRow = ({ thumbnail, title, dataPoints, onClick, children }: Toolbox
       <div className="toolbox-row__right">
         <span className="toolbox-row__right-title">{titleFromId(title)}</span>
         {dataPoints?.map((dataPoint) => (
-          <div key={dataPoint.text} className="toolbox-row__data-point">
+          <div key={dataPoint.tooltip} className="toolbox-row__data-point">
```

Comment:
tooltip isn't actually guaranteed to be unique.

Reply:
Added an id attribute to DataPoint